Localization/Policies/es-ES/DesktopAppInstaller.adml

```diff
@@ -95,6 +95,14 @@ Si desactiva esta directiva, no se pueden configurar fuentes adicionales para el
 Si habilita esta configuración, los usuarios podrán instalar paquetes de sitios web que usen este protocolo.
 
 Si deshabilita o no establece esta configuración, los usuarios no podrán instalar paquetes de sitios web que usen este protocolo.</string>
+      <string id="EnableWindowsPackageManagerCommandLineInterfaces">Habilitar Administrador de paquetes de Windows interfaces de línea de comandos</string>
```

Comment:
This one doesn't make much sense. It just says "Enable WPM CLI" but in Spanish that chain of nouns would need a connecting word to make sense, like "Enable CLI for WPM" or "Enable CLI of WPM".
But I don't think we have a way to add comments to the ADML, do we?